### ISSUE TITLE
Checkbox Tree makes admin UI unresponsive, slow

### ISSUE THREAD

**Issue by bcartier** [directus/directus#13460]
### Describe the Bug

When you have a checkbox tree with many nodes, it interferes with the responsiveness of the reset of the admin UI, particularly keyup events, but even focusing on any field is slow. 

We have a collection with two checkbox tree fields each with more than 200 nodes. Typing into a text or wysiwyg field on the same Item edit form, there can be a 4 to 5 second delay for text that you type to show up in the UI. This happens even if you don't interact with the tree fields. 

It seems like the checkbox tree interface is flooding the keyup event with anonymous function call or something. 

I've attached a screenshot of the performance tab from Chrome devtools  which is the result of typing a few characters into a WYSIWYG field. 

Removing the checkbox fields from the collection, or burying the checkbox fields in an accordion (keeping them out of the dom) immediately relieves the typing delays and slow-down. 

### To Reproduce

1. Create a collection with at least one Checkbox Tree field with a large number of nodes, and add some text and WYSIWIG fields 

2. Load the form and examine the devtool performance when simply typing into a text/rich text field. 

### Errors Shown

None, although Chrome dev tools reports every keypress event as a  "Long Task" in the performance tab.

### What version of Directus are you using?

9.10

### What version of Node.js are you using?

16.15

### What database are you using?

MySQL

### What browser are you using?

Chrome 101.0.4951.64. MacOS

### How are you deploying Directus?

Docker

**Comment by bcartier** [comment]
![Screen Shot 2022-05-20 at 5 26 37 PM](https://user-images.githubusercontent.com/48411/169622789-de2cc450-9979-4266-b2bc-d74cd074dccb.jpg)

**Comment by rijkvanzanten** [comment]
Similar to [#13237](https://github.com/directus/directus/issues/13237)

**Comment by bcartier** [comment]
Although this may be similar to #13237, in our case, there are only 38 fields and 4 field groups in the data model. 

Also worth mentioning - changing the 4 field groups from Detail Groups to Raw Groups nested in an Accordion seems to have reduced the impact. Using this as a work around for now, but having the groups with the checkbox trees open is still almost unusable because of input delays.

**Comment by rijkvanzanten** [comment]
Yeah! Not saying it's a duplicate or the same underlying problem, just making sure that we're aware that there's two different problems with similar symptoms 🙂
